resources/data/nrf_connect_sdk-0.1.3.json

```diff
@@ -49,6 +49,62 @@
                 }
             ]
         },
+        {
+            "type": "Checkable",
+            "steps": [
+                {
+                    "type": "Step",
+                    "platforms": [
+                        { "platform": "darwin" }
+                    ],
+                    "title": "Installing additional tools for Connected Home over IP applications development",
+                    "description": [
+                        "_Note: If you are not interested in the [Connected Home over IP](https://www.connectedhomeip.com/) applications, you can ommit steps presented in this section._",
+                        "Download and extract [GN](https://gn.googlesource.com/gn/) build system executable archive used by the Connected Home over IP to generate output files:",
+                        {
+                            "type": "commands",
+                            "description": [
+                                "cd ~",
```

Comment:
Just noticed that Getting started assistant is using `${HOME}` instead of `~`. Maybe follow the convention here?